README.md

```diff
@@ -467,3 +468,39 @@ They are as follows:
 [flipcoin code example]: https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/docs/example_pipelines/condition.py
 [installodh]: https://opendatahub.io/docs/quick-installation
 [kfp-tekton]: https://github.com/kubeflow/kfp-tekton
+
+# Deployment and Testing Guidelines for Developers
+
+To run the DSPO locally :
```

Comment:
We should also mention that you need to be logged-into the OpenShift cluster via `oc login`, and CRDs need to be installed (how do we do that?)